lib/src/lib.rs

```diff
@@ -222,9 +188,63 @@ impl Tester {
                     }
                 };
 
+                log::info!("Testing Actor {}", target.name);
+
                 let root = self.state_tree.flush();
+                let blockstore = self.state_tree.store().clone();
+                let mut executor = Executor::new(
+                    blockstore,
+                    root,
+                    self.builtin_actors.root,
+                    self.account.1,
+                    test_address,
+                    target_id.clone().into(),
+                );
+
```

Comment:
Couldn't we execute `Constructor` and `SetUp` before the first `flush()` and `Store::clone()`?